pyproject.toml

```diff
@@ -46,5 +47,12 @@ balanced_wrapping = false
 known_typing = "typing"
 sections = "FUTURE,STDLIB,TYPING,THIRDPARTY,FIRSTPARTY,LOCALFOLDER"
 
+[tool.ruff]
+target-version = "py38"
+line-length = 120
```

Comment:
IMO 120 is quite long, but isort was configured like this.

Reply:
For me 120 feels quite long too, generally I prefer 80 (or max. 100).

Reply:
I think 100 is the sweet spot.